draft-js-buttons/src/index.js

```diff
@@ -15,12 +17,18 @@ import AlignBlockDefaultButton from './components/AlignBlockDefaultButton';
 import AlignBlockCenterButton from './components/AlignBlockCenterButton';
 import AlignBlockLeftButton from './components/AlignBlockLeftButton';
 import AlignBlockRightButton from './components/AlignBlockRightButton';
+import SubButton from './components/SubButton';
+import SupButton from './components/SupButton';
 
 export {
   createBlockStyleButton,
   createInlineStyleButton,
+  createBlockAlignmentButton,
+  createOnClickButton,
```

Comment:
why are these two being exported here? Please remove

Reply:
creatOnClick button is used to create buttons with custom click events that are outside the "standard" toolbar implementations. Example usage: 

```
import React, { Component } from 'react'
import PropTypes from 'prop-types'
import insertTeXBlock from 'draft-js-katex-plugin/lib/modifiers/insertTeXBlock'

import asciimath2latex from 'asciimath-to-latex'

import createOnClickButton from './createOnClickButton'

export default createOnClickButton({
  onClick: ({ getEditorState, setEditorState }) => {
    setEditorState(insertTeXBlock(getEditorState(), asciimath2latex, 'int(x-2)', false))
  }
})
import React, { Component } from 'react'
import PropTypes from 'prop-types'
import insertTeXBlock from 'draft-js-katex-plugin/lib/modifiers/insertTeXBlock'

import asciimath2latex from 'asciimath-to-latex'

import createOnClickButton from './createOnClickButton'

export default createOnClickButton({
  onClick: ({ getEditorState, setEditorState }) => {
    setEditorState(insertTeXBlock(getEditorState(), asciimath2latex, 'int(x-2)', false))
  }
})
```

Reply:
hey @tarjej. Thanks! I think it's a good idea to add Sup/Sub buttons, but am not sure yet whether adding a generic button is a good idea and if so what the api of such a button should be. In any case it should be considered a different pr. So please remove it from this one :+1: